pkgs/by-name/go/go-bindata-assetfs/package.nix

```
@@ -2,17 +2,20 @@
   lib,
   buildGoModule,
   fetchFromGitHub,
+
+  go-bindata,
+  makeWrapper,
 }:
 
 buildGoModule rec {
   pname = "go-bindata-assetfs";
-  version = "unstable-2022-04-12";
+  version = "unstable-2025-02-01";
```

```suggestion
  version = "0-unstable-2025-02-01";
```
Since recently all versions must start with a digit